docs/x509/ocsp.rst

```diff
@@ -612,6 +621,14 @@ Interfaces
 
         The single extensions encoded in the response.
 
+    .. attribute:: responses
+
+        .. versionadded:: 37.0.0
+
+        :type: :class:`~cryptography.x509.ocsp.OCSPResponseIterator`
```

Comment:
Instead of declaring this to be `OCSPResponseIterator`, just say "an iterator of OCSPSingleResponse".